algebras/testkit/src/main/scala/endpoints4s/algebra/client/EndpointsTestSuite.scala

```diff
@@ -13,7 +13,7 @@ trait ClientEndpointsTestApi extends EndpointsTestApi {
 
 trait EndpointsTestSuite[T <: ClientEndpointsTestApi] extends ClientTestBase[T] {
 
-  def clientTestSuite() = {
+  def clientTestSuite(timeoutTest: Boolean = true) = {
```

Comment:
The test is not supported by akka-http yet

Reply:
The difference between the ChunkedJsonEntities tests and the Client tests is that the Client tests are encapsulated in a method, whereas the ChunkedJsonEntities tests are implement directly within the trait.

Making this test modular using cake pattern would require to inject a new method and calling it in each test suite supporting the timeout settings. Would that be a solution or should we keep the boolean parameter until all clients implement the timeout setting?

Reply:
I don’t think we need the extra method in the trait (even that’s what in most of the existing test suites, I am not sure this provides value). I recommend doing something like this:

https://github.com/endpoints4s/endpoints4s/blob/3545ad7c118b73bf6f44cc0fefd1b08b2804953d/algebras/testkit/src/main/scala/endpoints4s/algebra/client/ChunkedEntitiesTestSuite.scala#L12

Reply:
I would prefer if you could move the test to a separate trait that you would mix only to the relevant interpreters (instead of using the boolean param). Would you mind making that change?